site/content/docs/5.2/components/progress.md

```diff
@@ -36,19 +40,9 @@ Put that all together, and you have the following examples.
 </div>
 {{< /example >}}
 
-{{< callout info >}}
-### Changed markup structure for progress bars
+## Sizing
 
-Note that the recommended markup for progress bars has changed compared to past versions of Bootstrap. Previously, the `role` and various `aria-` attributes were on the inner `.progress-bar` element.
-
-```html
-<div class="progress">
-  <div class="progress-bar" role="progressbar" aria-label="Basic example" aria-valuenow="0" aria-valuemin="0" aria-valuemax="100"></div>
-</div>
-```
-
-The problem with this structure is that progress bars with a zero value are completely ignored, and not announced, by assistive technologies. While the legacy structure will still be displayed correctly, we strongly recommend updating to the new structure.
-{{< /callout >}}
+### Width
```

Comment:
I understand why this is modified like that but I've found it weird when I read it to have a "Sizing > Height" section that explains how to change the height of the entire progress bar and a "Sizing > Width" section that doesn't explain how to reduce the width of the progress bar but rather its content; its value.

Reply:
I can see what you mean, but in a sense both are about sizing the width and height of the bars themselves...just that the height is set for the whole of the progress element. Maybe changing the first heading to "Bar sizing"?

Reply:
Made a tweak to the sizing section @julien-deramond @mdo to make it clearer

Reply:
OK for me 👌